RedHat-UX/red-hat-design-system
# docs: added styles for list item line height

## What I did

1. Modified `_base.scss` to add li line-height set at 2 in order to close #1025.

## Testing Instructions

1. Go to [any list on DP](https://deploy-preview-1026--red-hat-design-system.netlify.app/elements/tooltip/guidelines/#tooltip-vs.-popover).
2. Inspect a list item.
3. Line height should be 2.
